gammapy/time/plot.py

```diff
@@ -30,3 +28,140 @@ def plot_time_difference_distribution(time, ax=None):
 
     # TODO: implement!
     raise NotImplementedError
+
+def plot_light_curve(name_3FGL, time_start, time_end, ax = None):
+    """Plot flux as a function of time for a fermi 3FGL object.
+
+    Parameters
+    ----------
+    name_3FGL : `string`
+        The 3FGL catalog name of the object to plot
+    fermi_met_start : `~astropy.time.Time`
+        Astropy time object for the start of the light curve
+    fermi_met_end : `~astropy.time.Time`
+        Astropy time object for the end of the light curve
+    ax : `~matplotlib.axes.Axes` or None
+        Axes
+    Returns
+    -------
+    ax : `~matplotlib.axes.Axes`
+
+    Examples
+    --------
+    Plot effective area vs. energy:
+
+    .. plot::
+        :include-source:
+
+        import gammapy.time
+        import astropy.time
+        import matplotlib.pyplot as plt
+
+        time_start = astropy.time.Time('2010-01-01T00:00:00')
+        time_end = astropy.time.Time('2015-02-02T02:02:02')
+
+        plt.plot = gammapy.time.plot_light_curve('3FGL J0349.9-2102', time_start, time_end)
+        plt.show()
+    """
+    from ..datasets import fetch_fermi_catalog
+    import astropy.time
+    import numpy as np
+    import matplotlib.pyplot as plt
+    import matplotlib.dates as mdates
+    import math
+
+    ax = plt.gca() if ax is None else ax
+
+    fermi_met_base = astropy.time.Time('2001-01-01T00:00:00')
+
+    fermi_met_start = (time_start - fermi_met_base).sec
+
+    fermi_met_end = (time_end - fermi_met_base).sec
+
+    # As far as I know light curves were only included in 3FGL, so we must use this catalog.
```

Comment:
I think earlier Fermi catalogs also had light curves. Can you please check this?
There's probably little value in adding an option to plot light curves from earlier catalogs (except for checking analysis errors in older catalogs, could be fun), so I'm not saying you should make this an option. Either making this an option or updating this comment would be OK.